backends/memcache.go

```diff
@@ -173,3 +173,14 @@ func (m *Memcache) SRem(key string, value string) (bool, error) {
 	return false, errors.New("mantle: SETS/SRem unimplemented for memcache datastore")
 }
 
+// Raise unimplemented error
+func (m *Memcache) Sismember(key string, member string) (bool, error) {
+	return false, errors.New("mantle: SETS/Sismember unimplemented for memcache datastore")
+}
+
+func (m *Memcache) Sismembers(key string, members []string) ([]bool, error) {
```

Comment:
Hey Harshad,

I had couple of concerns and wanted to know your opinions. 
1. Do we need Sismembers?. Just like Sadd, we can use Sismember to work with multiple keys IMO.
2. Instead of returning NoImplemented error, wouldnt it be useful if we could use "get" or any other workaround to check membership?.